Caboodle/SecureStorage/SecureStorage.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if !NETSTANDARD1_0
+using System.Security.Cryptography;
+#endif
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class SecureStorage
+    {
+        // md5(Caboodle.SecureStorage)
+        const string localDirName = "d2f74ffc823f1d9f015ee8edf1463498";
+
+        public static Task<string> GetAsync(string key)
+            => PlatformGetAsync(Md5Hash(key));
```

Comment:
In order to get a valid filename I'm just hashing the given key (so it doesn't matter what the key contains).  Thoughts?